### DISCUSSION TITLE
How to rotate game world in phaser3?

### DISCUSSION THREAD

**Discussion by monkeyWie** [photonstorm/phaser#6076] [Q&A]
If user's phone is portrait, the game rotate 90deg, and fill the screen (width < height). The user will rotate their phone.
If user's phone is landscape. the game show as normal.

Like this game:
![image](https://user-images.githubusercontent.com/13160176/163183349-78183819-0d8d-40ff-ac5d-69d97598f525.png)

![image](https://user-images.githubusercontent.com/13160176/163183360-007b7114-d2d1-4da1-9757-8bd0f4bf3ca0.png)

**Comment by spayton** [comment]
Rotating the camera would be easiest, see: [https://phaser.io/examples/v3/view/camera/rotate-camera](url)

or add everything to a base container and rotate that, this might incur extra position correction.

You might have trouble with masking, they can operate in world space so won't move with camera/container.
